content/c-sharp/concepts/strings/terms/format/format.md

```diff
@@ -0,0 +1,234 @@
+| `f` to `fffffff` | The fraction of a second: tenth (`f`), hundredths (`ff`), ..., or ten millionths (`fffffff`).                             | `HH:mm:ss.fff` >> `20:05:09.000`   |
+| `F` to `FFFFFFF` | If non-zero, the fraction of a second: tenth (`F`), hundredths (`FF`), ..., or ten millionths (`FFFFFFF`).                | `HH:mm:ss.fFF` >> `20:05:09.0`     |
+| `t`, `tt`        | The first character of the AM/PM designator (`t`) or the entire one (`tt`).                                               | `hh:mm t` >> `08:05 P`             |
+| `\`              | Escape character to display characters otherwise interpreted as specifiers. It must be doubled in quoted string literals. | `\da\y: d` >> `day: 7`             |
+| any other        | Displayed as is.                                                                                                          | `MM-dd-yyyy` >> `06-07-2009`       |
+
+The characters used as separators and other symbols in the formatted date or time depend on the culture specified.
+
+## Enum Formats
+
+There are only standard formats available to format enum values, namely `G`, `F`, `D`, and `X`.
+
+For example, let the `Access` type be defined as:
+
+```cs
+public enum Access { Read = 1, Write = 2};
+```
+
+With the `G` specifier:
+
+- An enum value is displayed as its enum name. For example `Access.Read` is displayed as `Read`.
+- A combination of enum values ​​using the bitwise OR operator (`|`) is displayed as a comma-separated list of matching names. For example `Access.Read | Access.Write` is displayed as `Read, Write`.
+- An integer (cast to an enum type) is displayed as the name of the matching enum value if found. For example `((Access)1)` is displayed as `Read` while `((Access)3)` is displayed as `3` since there is no matching enum name. But if the enum type is defined with a `[Flags]` attribute (e.g., `[Flags] public enum Access { ... };`), an attempt is made to find a list of matching names. In that case, `((Access)3)` would be displayed as `Read, Write` since `1 | 2 = 3`.
+
+> **Note:** To understand why `1 | 2 = 3`, convert the operands in binary format (`1` becomes `01` and `2` becomes `10`), then apply the `|` operator bit by bit using the following rules `0 | 0 = 0`, `0 | 1 = 1`, `1 | 0 = 0`, and `1 | 1 = 1`. The result of the operation `10 | 01` is `11` which corresponds to `3`.
+
+The `F` specifier works like `G`, but it attempts to display an integer as a list of matching names even if the enum type is not defined with the `Flags`attribute. For example, in our case `((Access)3)` is displayed as `Read, Write` even if the `Access` type is defined without the `[Flags]` attribute.
+
+The `D` displays the enum value as the corresponding integer, and `X` as the corresponding hexadecimal value. For example, `Access.Write` is displayed as `2` with `D`, and as `00000002` with `X`.
+
```

Comment:
Please remove this section. The content is good, but for a term entry, it is too big.